linodecli/configuration.py

```diff
@@ -63,7 +63,7 @@ def update(self, namespace, username=None):
         if not username:
             username = "DEFAULT"
 
-        if not self.config.has_option(username, 'token'):
+        if not self.config.has_option(username, 'token') and not os.environ.get(ENV_TOKEN_NAME, None:
```

Comment:
pretty sure you're missing a `)` here

Reply:
Nice catch. I must've accidentally removed it right before committing :S I'll update the PR.

Sorry about that.